src/content/en/2021/caching.md

```diff
@@ -17,4 +19,307 @@ featured_stat_3: TODO
+
+What is or would be the impact from using expires more effectively?
+
+Could we possibly see big wins from getting more consistent here?
+
+### Vary
+
+Talk about Vary adoption, 42% of requests on mobile set Vary: accept-encoding.
+
+In addition, 83% of mobile Vary: accept-encoding requests also use cache-control. This is another case of the more you know about caching, the more consistently you can make use of all the resources available to you.
+
+The most common Vary headers are: `accept-encoding`, `user-agent`, `origin`, `accept`.
+
+### TTL adoption
+
+Talk about 28% of mobile requests use neither Cache-Control nor Expires headers.
+
+The median HTML resource on mobile has a TTL of 14 days
+
+## Cookie Cache adoption
+
+35.63% of Cacheable Desktop pages with set-cookie
+Mobile pages with set-cookie: 35.99%
+
+Another area of opportunity we discovered was with cacheable pages that are not using `set-cookie`. About 35% of Desktop and Mobile pages use `set-cookie` while over 64% of them do not.
+
+Large area of opportunity for pages that are actually cacheable, but are just missing this directive?
+
+Fun fact `test_cookie` is the most popular desktop and mobile cookie name at 4% of all cookies.
+
+{{ figure_markup(
+  caption="TODO",
+  content="4%",
+  classes="big-number",
+  sheets_gid="455647224",
+  sql_file="todo.sql"
+) }}
+
+The top cookies attributes used on Desktop and Mobile are SameSite and Secure. SameSite is on 53% of cookies set in Desktop responses and a whopping 68% on Mobile responses.
+
+Why is it a big deal that SameSite is at 68% on mobile?
+
+## What type of content are we caching?
+
+Talk about what kind of resources are we caching, (resource age groups: party, type).
+
+The median age of HTML resources cached for mobile is 9 weeks, while for text the median age is 1220 weeks (23.4 years).
+
+Talk about, total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about 54% of mobile resources are older than their TTL.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about, 60% of 1P mobile resources are older than their TTL, which is more common than 3P resources (45%).
+
+Explain why this is significant- 60% of 1st Party mobile resources are older than their TTL- is this because 1P are more important or crucial? Because 3P are being deprecated?
+
+## How do cache TTLs compare to resource age?
+
+The median difference between resource TTL and resource age is -6 days, meaning the age is 6 days older than the TTL.
+
```

Comment:
Mobile resources with a cache TTL that was considered too short compared to its content age have seen an improvement since 2020. 60.2% in 2020 but now 54% in 2021. 
Developers are getting better at setting the cache duration more accurately to the content age, resulting in more responsible caching.

Reply:
When we split the data between first party and third party providers, the largest improvements come from third party where we have a 13.2% improvement. It is highly encouraging to see how companies around the world are building products for developers that are highly optimized on caching. However, the challenge remains for how first parties can effectively improve over the coming years. 

The community's increased attention on improving performance has encouraged and even incentivized third parties to optimize their caching strategies.